src/clientiface.cpp

```diff
@@ -250,7 +267,9 @@ void RemoteClient::GetNextBlocks (
 			*/
 
 			f32 dist;
-			if (!isBlockInSight(p, camera_pos, camera_dir, camera_fov, d_blocks_in_sight, &dist)) {
+			if (!(isBlockInSight(p, camera_pos, camera_dir, camera_fov, d_blocks_in_sight, &dist) ||
+				(playerspeed.getLength() > 1.0 * BS &&
```

Comment:
1.0f

Reply:
Yes please.

Reply:
Arrgggghhh. I must have removed the f by accident.

Reply:
Done